app/model/editions/templates/TemplateHelpers.scala

```diff
@@ -81,26 +81,66 @@ object SpecialEditionButtonStyles {
   implicit val formatSpecialEditionButtonStyles : OFormat[SpecialEditionButtonStyles] = Json.format[SpecialEditionButtonStyles]
 }
 
-trait EditionDefinition {
+/**
+  * A curated product that has a push publication model – it
+  * pushes updated content to the world in discrete 'editions' that contain a
+  * list of fronts.
+  *
+  * Contrast with e.g. web/app Fronts, which publish their content in real time
+  * on a container by container basis.
+  */
+trait CuratedPlatformDefinition {
   val title: String
   val subTitle: String
   val edition: String
-  val header: Header
-  val editionType: EditionType
   val notificationUTCOffset: Int
-  val topic: String
   val locale: Option[String]
+  val platform: CuratedPlatform
+}
+
+object CuratedPlatformDefinition {
+  implicit def formatCuratedPlatform: OWrites[CuratedPlatformDefinition] = {
+    case editionsApp: EditionsAppDefinition =>
+      EditionsAppDefinition.formatEditionDefinition.writes(editionsApp)
+    case genericApp: CuratedPlatformWithTemplate =>
+      CuratedPlatformWithTemplate.writes.writes(genericApp)
```

Comment:
It might be nicer to find this value implicitly (and formatEditionDefinition)? Or if not, why is `def writes` marked implicit?

Reply:
Yeah, that's nicer: [c339dbc](https://github.com/guardian/facia-tool/pull/1557/commits/c339dbc9fb3b192b44935415c1559c89d643010d) 👍